uwrt_mars_rover_utils/uwrt_mars_rover_xbox_controller/include/uwrt_mars_rover_xbox_controller/visibility.h

```diff
@@ -0,0 +1,65 @@
+// Copyright 2016 Open Source Robotics Foundation, Inc.
```

Comment:
I know I added this and it's for windows builds but.. I think it just clutters the repo and we should remove all visibility.h files. Thoughts @keyonjerome